baseplate/lib/config.py

```diff
@@ -379,6 +389,30 @@ def tuple_of(text: str) -> Sequence[T]:
     return tuple_of
 
 
+def DefaultFromEnv(
+    item_parser: Callable[[str], T], default_src: str
+) -> Callable[[str], OptionalType[T]]:  # noqa: D401
+    """An option of type T or a default.
+
+    The default is sourced from an environment variable with the name ``default_src``.
+    Either the option or the default must be provided
+    """
+    default = None
+    env = os.getenv(default_src)
+    if env:
+        default = Optional(item_parser)(env)
+
+    def default_from_env(text: str) -> OptionalType[T]:
+        if text:
+            return Optional(item_parser)(text)
+        if default:
```

Comment:
tiny nit: can you put a blank between these two separate if branches so they don't read at a distance like they're if/elif pairs please?